tests/Feature/Offers/AppointmentsTest.php

```
@@ -65,6 +73,18 @@ protected function setUp(): void
             'title'       => 'test',
             'color'       => '#FFFFFF',
         ]);
+
+        $this->appointment = Appointment::factory()->create([
+            'user_id'  => $this->user->id,
+            'start_at' => Carbon::now(),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Carbon\Carbon' in method 'setUp'. (undefined)

(StaticAccess)